src/vs/workbench/browser/parts/statusbar/statusbarPart.ts

```diff
@@ -287,37 +286,20 @@ class StatusBarEntryItem implements IStatusbarItem {
 	private executeCommand(id: string, args?: any[]) {
 		args = args || [];
 
-		// Lookup built in commands
-		const builtInActionDescriptor = Registry.as<IWorkbenchActionRegistry>(ActionExtensions.WorkbenchActions).getWorkbenchAction(id);
-		if (builtInActionDescriptor) {
-			const action = this.instantiationService.createInstance(builtInActionDescriptor.syncDescriptor);
-
-			if (action.enabled) {
-				/* __GDPR__
-					"workbenchActionExecuted" : {
-						"id" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" },
-						"from": { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
-					}
-				*/
-				this.telemetryService.publicLog('workbenchActionExecuted', { id: action.id, from: 'status bar' });
-				(action.run() || TPromise.as(null)).done(() => {
```

Comment:
@bpasero What's up with this `workbenchActionExecuted`-event? Is its purpose really to only report *workbench*-actions? So no, editor command, no extension commands?

Reply:
@jrieken I would think that this telemetry event is logged for a) workbench actions, b) editor actions and c) contributed actions. The name is probably old and we never changed it but the data should be that.

Reply:
Great. In that case this was a bug.